src/components/HomepageFeatures/index.js

```diff
@@ -51,19 +48,49 @@ function Feature({ Svg, title, description }) {
 export default function HomepageFeatures() {
   return (
     <section>
-      <h2 className={styles.featuresTitle} id="features">
-        A quick glance at our features 👀
-      </h2>
-
-      <div className={styles.features}>
-        <div className="container">
-          <div className="row">
-            {FeatureList.map((props, idx) => (
-              <Feature key={idx} {...props} />
-            ))}
-          </div>
-        </div>
+      <div className="mt-6 grid grid-cols-1 gap-x-6 gap-y-10 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-3 mb-16">
+        {FeatureList.map((props, idx) => (
```

Comment:
In my opinion should be renamed to camelCase, because it confuses me if it is a component or an array